### ISSUE TITLE
State icon margin/padding is incorrect

### ISSUE THREAD

**Issue by richard-cox** [rancher/dashboard#8389]
**Setup**
- Rancher version: `master` branch (2.7 Q2). 
- Browser type & version:

**Describe the bug**
- There's been a regression in padding after the merge of nuxt removal in padding/margin
- The status icon afterwards is flush with text
- This could be a rebase issue?

**We need to understand what has caused this and any other place that is affected. Fixing this one location is not the best approach**

Could have been caused by nuxt PR or something that has merged since branching

**To Reproduce**
- Navigate to a cluster detail page

**Result**
- See screesnhots

**Screenshots**
`latest`
![image](https://user-images.githubusercontent.com/18697775/224034596-4a85d27d-a8a2-475f-87fb-b74311fea2a9.png)

`release-2.7.2`
![image](https://user-images.githubusercontent.com/18697775/224034342-f2f764e9-7750-40f9-b664-4505576c7362.png)

**Comment by nwmac** [comment]
How does this stuff keep getting broken? Did this change due to the nuxt removal PR? @richard-cox

**Comment by richard-cox** [comment]
@nwmac Not sure, could have been nuxt or another PR merged since we branched. We should definitely understand what caused it, because I'd bet there's other affected places.

**Comment by richard-cox** [comment]
@bisht-richa part of this issue involves understanding how the regression occurred. can you add a comment here with your findings?

**Comment by richard-cox** [comment]
Adding more places that probably suffered the same bug.

1. delete / edit buttons bottom right of cluster tools chart card
  ![image](https://user-images.githubusercontent.com/18697775/232727440-378b593a-db47-4b76-a9bb-cf248bc8789b.png)

**Comment by bisht-richa** [comment]
I did some tests and there is a setting in Nuxt config to preserve whitespace which is by default `true` that's why we can see those spaces between some elements.

but if we add this code in Nuxt config and set `preserveWhitespace` to false it will disables the space

 ```js
loaders: {
        vue: {
          compilerOptions: {
            preserveWhitespace: false
          }
        }
      },
    ```

But as we don't have Nuxt.js in the application it is causing a missing space issue.

https://zube.io/files/rancher/5cc9052e7639e8b46b1d7231c167a5f4-screen-recording-2023-04-25-at-15-44-20.mov

**Comment by nwmac** [comment]
@bisht-richa Awesome find - this is indeed the issue - I have put up a PR that updates the vue config to patch the webpack config for the view loader - https://github.com/rancher/dashboard/pull/8742

**Comment by richard-cox** [comment]
@bisht-richa This one should be fixed via #8742. Could you also double check the other places listed here are fixed?

**Comment by bisht-richa** [comment]
@richard-cox Yes I checked all the other places everything is fixed.
